harper-core/tests/text/linters/Alice's Adventures in Wonderland.snap.yml

```diff
@@ -830,6 +874,16 @@ Suggest:
 
 
 
+Lint:    Miscellaneous (0 priority)
+Message: |
+     412 | advisable to go with Edgar Atheling to meet William and offer him the crown.
+         |                                             ^~~~~~~ Consider adding an article before this noun.
```

Comment:
This definitely is different. Proper nouns are already "definite" and so don't take articles unless it's part of the name like in "The Gambia" or surnames when referring to the whole family like in "The Simpsons".
And of course in a humorous style: "I met two Toms and a William".